vsintegration/src/FSharp.Editor/FSharp.Editor.fsproj

```diff
@@ -93,6 +96,8 @@
   </ItemGroup>
   <ItemGroup>
     <Reference Include="mscorlib" />
+    <Reference Include="PresentationFramework" />
+    <Reference Include="WindowsBase" />
```

Comment:
PresentationFrameword is needed for `Label` type.
WindowsBase is for `FrameworkElement`.